jsonnet/rules.jsonnet

```diff
@@ -186,6 +186,16 @@
               severity: 'warning',
             },
           },
+          {
+            expr: 'cluster:alertmanager_routing_enabled:max > 0',
+            alert: 'AlertmanagerReceiversNotConfigured',
+            annotations: {
+              message: 'Alerts are not configured to be sent to a notification system. Configuring an alert route will allow you to be notified when important failures occur.',
```

Comment:
As the outcome, form my understanding, is for users to configure these routes, should we provide more information here?

Reply:
Example?

Reply:
Agree on this -> https://github.com/openshift/cluster-monitoring-operator/pull/585#discussion_r359416714